examples/api-dotenv-request/app/Main.hs

```diff
@@ -0,0 +1,31 @@
+import Configuration.Dotenv (loadFile, defaultConfig)
+import System.Environment (lookupEnv)
+import Network.HTTP.Req
+import qualified Data.ByteString.Char8 as B
+import qualified Data.Text as T
+import Data.Aeson
+
+main :: IO ()
+main = do
+    loadFile defaultConfig
+    
+    --  Loading API URI from dotenv 
+    maybeApiEndpoint <- lookupEnv "API_URI"
+
+    case maybeApiEndpoint of
+        Just apiEndpoint -> do
```

Comment:
Can you replace this `case` statement with the [`maybe`](https://hackage.haskell.org/package/base-4.19.0.0/docs/Prelude.html#v:maybe) function?
```haskell
    maybe
        (putStrLn "API_ENDPOINT env...")
        (\apiEndpoint -> do
             ....
        )
        maybeApiEndpoint
```